src/pages/index.tsx

```diff
@@ -1,45 +1,74 @@
-import type { GetStaticProps, InferGetStaticPropsType } from 'next'
+import { GetStaticProps, InferGetStaticPropsType } from 'next'
 import { useLiveQuery } from 'next-sanity/preview'
 
 import Card from '~/components/Card'
-import Container from '~/components/Container'
+import Navigation from '~/components/Navigation'
+import Hero from '~/components/Hero'
 import Welcome from '~/components/Welcome'
 import { readToken } from '~/lib/sanity.api'
 import { getClient } from '~/lib/sanity.client'
-import { getPosts, type Post, postsQuery } from '~/lib/sanity.queries'
+import {
+  getPosts,
+  type Post,
+  postsQuery,
+  navbarQuery,
+  getNavbarData,
+  getHeroData,
+  heroQuery,
+} from '~/lib/sanity.queries'
+import { NavigationI } from '~/lib/interfaces'
 import type { SharedPageProps } from '~/pages/_app'
 
 export const getStaticProps: GetStaticProps<
   SharedPageProps & {
     posts: Post[]
+    navbarItems: NavigationI[]
   }
 > = async ({ draftMode = false }) => {
-  const client = getClient(draftMode ? { token: readToken } : undefined);
-  const posts = await getPosts(client);
-
+  const client = getClient(draftMode ? { token: readToken } : undefined)
+  const posts = await getPosts(client)
+  const navbarItems = await getNavbarData(client)
+  const heroItems = await getHeroData(client)
   return {
     props: {
       draftMode,
       token: draftMode ? readToken : '',
       posts,
+      navbarItems,
+      heroItems
     },
   }
 }
 
-export default function HomePage(
-  props: InferGetStaticPropsType<typeof getStaticProps>,
-) {
-  const [posts] = useLiveQuery<Post[]>(props.posts, postsQuery);
+export default function HomePage({
+  posts,
+  navbarItems,
+  heroItems,
+}: InferGetStaticPropsType<typeof getStaticProps>) {
+  const [postsData] = useLiveQuery<Post[]>(posts, postsQuery)
+  const [navbarItemsData] = useLiveQuery(navbarItems, navbarQuery)
+  const [heroItemsData] = useLiveQuery(heroItems, heroQuery)
+
+  const navbarItem = navbarItemsData?.[0] || {}
```

Comment:
you can load first from query instead https://github.com/julia-dizhak/uawit-website/pull/8